linuxrc.c

```diff
@@ -1278,6 +1278,13 @@ void lxrc_init()
 
   util_update_kernellog();
 
+  if(config.loghost)
+  {
+    char cmd[40];
```

Comment:
40 bytes buffer seems to be quite a small buffer to me, I guess when user uses a long FQDN it might not fit there. I'd use at least 128 byte.

Moreover you should check the `snprintf` result and not run the command when there is not enough space in the buffer (the command is truncated).